views/customize-list.handlebars

```diff
@@ -48,7 +48,12 @@
                             <div>
                                 <label for="{{this.obj}}" class="pure-checkbox">
                                 <input class="dayCheckbox" id="{{this.obj}}" type="checkbox" value="{{this.obj}}" onchange="refreshArtistList()" {{state}}>
-                                {{this.obj}}</label>
+                                {{#if ../dates.length}}
+                                    {{formatDate (lookup ../dates @index)}}
```

Comment:
> This is kind of annoying and I apologise for it but I hate how this displays dates by default haha. Is there a way to pass a format string to it to make it look nicer? Or do we have to have our own formatDate handler registered in the code to do that

How about "Thursday July 6th" instead of "Thu Jul 06 2023"? Year can be dropped because it's already at the top of the page.